portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,149 @@
+    mainText.classList.add("eerie-text"); 
+    //the following lines display extra, EM-exclusive features 
+    extraContent.style.display = "block"; 
+    extraContent.style.opacity = "1";  
+    //TODO: fix below line, which doesn't transition the opacity
+    extraContent.classList.add("change-extra-content-opacity"); 
+    secretText.textContent = "Back"; 
 }
+
+function deactivateEerieMode() {
+    eerieModeOn = false; 
+    mePic.classList.remove("fade-image-out"); 
+    document.body.style.backgroundColor = "white"; 
+    root.style.setProperty('--target-color', "black");
+    root.style.setProperty('--circle-cursor-color', "black"); 
+    root.style.setProperty('--highlighted-text-color', "white"); 
+    mainHeading.classList.add("eerie-text"); 
+    sections.forEach(section => {
+         section.classList.remove("eerie-text"); 
+    });
+    mainHeading.textContent = "Kira's Profile"; 
+    mainText.textContent = "Wasn't that fun?"; 
+    mainText.classList.remove("eerie-text"); 
+    secretText.textContent = "???"; 
+    //the following lines disable extra, EM-exclusive features 
+    extraContent.style.display = "none"; 
+}
+
+/**
+ * Animates the mouse as it interacts with section headers. 
+ */
+const sections = document.querySelectorAll(".sections li");
+sections.forEach(section => {
+    section.addEventListener("mouseover", e => {
+        circleAroundMouse.classList.add("grow-cursor");
+        if (eerieModeOn) { section.classList.remove("eerie-text"); } 
+        section.classList.add("color-text");  
+    }); 
+    section.addEventListener("mouseleave", e => {
+        circleAroundMouse.classList.remove("grow-cursor");
+        if (eerieModeOn) { section.classList.add("eerie-text"); } 
+        section.classList.remove("color-text"); 
+    }); 
+    section.addEventListener("click", e => {
+        if (section.id === "secret-text") {
+            mainText.textContent = secretMessages[messageIndex]; 
+            if (messageIndex < secretMessages.length) {
+                messageIndex += 1; 
+            } else {
+                if (!eerieModeOn) { activateEerieMode(); }
+                else {deactivateEerieMode(); }
+            }
+        } else {
+            mainText.textContent = section.dataset.message; 
+        }
+    })
+});
+
+document.addEventListener('keydown', function(event) {
+    if(event.keyCode == 39) {
+        activateEerieMode(); 
+    }
+    if(event.keyCode == 37) {
+        deactivateEerieMode(); 
+    }
+});
+
+/**
+ * Recommends a movie based on dropdown input 
+ */
+const dropdown = document.querySelector("#subgenres"); 
+let userSelectedValue; 
+
+spnMovies = ["Hereditary", "It Follows", "The Exorcist", "Ju-On: The Grudge", "Ringu"]
+nonSpnMovies = ["Midsommar", "The Shining", "Hush", "The Strangers", "Orphan"]
+
+const result = document.querySelector("#result");
+function recommendMovie() {
+    userSelectedValue = dropdown.options[dropdown.selectedIndex].value; 
```

Comment:
Check out the "this" operator... one of the more unique and confusing things in JavaScript but is the standard way of doing this rather than looking at an item on the page.